pkg/nsx/services/vpc/builder.go

```diff
@@ -69,7 +68,7 @@ func buildNSXVPC(obj *v1alpha1.VPC, nc VPCNetworkConfigInfo, cluster string, pat
 			},
 		}
 		vpc.SiteInfos = siteInfos
-		vpc.LoadBalancerVpcEndpoint = &model.LoadBalancerVPCEndpoint{Enabled: &VPCLBEndpointEnabled}
+		vpc.LoadBalancerVpcEndpoint = &model.LoadBalancerVPCEndpoint{Enabled: &nc.LoadBalancerVPCEndpointEnabled}
```

Comment:
Do we still need to store `LoadBalancerVPCEndpointEnabled`?

Reply:
for future extension, i just don't want to use a hardcode value "true" here, and use the value from networkconfig CR, in the CR parse part, i set its value to true, if we want to extend the param here, we can just change the mapping part without changing the code here.